recipe/meta.yaml

```
@@ -10,11 +10,13 @@ source:
   sha256: 0412fc77e364226c9395e3188a2d0ac5888317fecfba7bc2dbe003c2ddb5b3d3
 
 build:
-  number: 1
+  number: 0
   skip: true  # [not win]
 
 requirements:
   build:
+    # Use msys2 7zip to avoid a circular dependency between ucrt and 7zip on Windows.
+    - m2-p7zip  # [win]
```

```suggestion
    - msys2-p7zip  # [win]
```
Can you try a modern `msys2-p7zip` instead at the build time?

If I understand it well, you mean `7zip 24.09` ([feedstock](https://github.com/AnacondaRecipes/7zip-feedstock/blob/master/recipe/meta.yaml)) on win-64 conflicts with `ucrt`?

---

Yes, `7zip` will depend on `ucrt` via `vc` once the vc-feedstock release happens.

---

Either `m2-p7zip` or `msys2-p7zip` seem to work. Is one preferred over the other?

---

m2-p7zip is obsolete and can be replaced with  msys2-p7zip